### PR TITLE
Fix rhel-atomic image refresh

### LINKED ISSUE #180: Image refresh for rhel-atomic
Image refresh for rhel-atomic
 * [ ] FAIL: image-refresh rhel-atomic

### PR DESCRIPTION
**Pull request by martinpitt**
It seems our subscription once again doesn't work any more, so let's take the opportunity to drop it as a requirement entirely.

 * [x] image-refresh rhel-atomic

Fixes #180

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh in progress on 4-cockpit-9.
Log: https://209.132.184.41:8493/logs/image-refresh-278-20191125-212424/

**Comment by cockpituous** [comment]
image-refresh in progress on 4-cockpit-10.
Log: https://209.132.184.41:8493/logs/image-refresh-278-20191125-212424/

**Comment by martinpitt** [comment]
And yay, a log that is 8 kB instead of 1.2 MB of endless download progress :+1:

**Comment by cockpituous** [comment]
image-refresh rhel-atomic done: https://github.com/cockpituous/bots/commits/image-refresh-rhel-atomic-20191125-213131

**Comment by martinpitt** [comment]
Meh, so *that's* why we need the new libssh4..

https://209.132.184.41:8493/logs/pull-278-20191125-213333-a7271dc5-cockpit-project-cockpit-rhel-7.8-rhel-atomic/log.html#173-2

So it seems this is either blocked until our subscription gets fixed, or we somehow fix cockpit-dashboard in the rhel-7.8 branch to actually get along with the old version.

**Comment by cockpituous** [comment]
image-refresh in progress on 4-cockpit-10.
Log: https://209.132.184.41:8493/logs/image-refresh-278-20191126-070101/

**Comment by martinpitt** [comment]
I sent a ticket about our team subscription, but I also reworked this PR to not require it any more in the first place.

**Comment by cockpituous** [comment]
image-refresh rhel-atomic done: https://github.com/cockpituous/bots/commits/image-refresh-rhel-atomic-20191126-070808

File: images/scripts/lib/atomic.setup [hunk 1]
```diff
@@ -35,21 +35,10 @@ fi
...
-rm -f /etc/yum.repos.d/*
-cat >/etc/yum.repos.d/deps.repo <<EOF
-[deps]
-baseurl=file:///root/rpms
-enabled=1
-EOF
+# package overlay. Use the EPEL package on RHEL Atomic as well, so that we
+# don't need a subscription.
+mkdir -p /root/rpms
+(cd /root/rpms && curl --silent --show-error -O https://dl.fedoraproject.org/pub/epel/7Server/x86_64/Packages/l/libssh-0.7.1-0.7.el7.x86_64.rpm)
```
Comment (marusak) [inline images/scripts/lib/atomic.setup]: so we are gonna be stuck with this one version? And also wouldn't this disappear when new version is released? Koji should be safer https://koji.fedoraproject.org/koji/buildinfo?buildID=1240987

Comment (martinpitt) [reply images/scripts/lib/atomic.setup]: I'm actually counting on this to fail when there ever is a new RHEL Atomic update (it seems it's not going to be support for that long any more). If there ever is, we need to update the version there -- we should at least keep up with what's in EPEL.

**Comment by martinpitt** [comment]
> So not even rhel-7.7 needs subscription to be set up?

No, we haven't used subscriptions for any image except rhel-atomic for quite some time now.

**Pull request merged**